### ISSUE TITLE
Warn about min batch fee

### ISSUE THREAD

**Issue by alfetopito** [gnosis/dex-react#1227]
Reference https://github.com/gnosis/dex-services/issues/1099

Discussion https://gnosisinc.slack.com/archives/CSGLUC1BN/p1594629702005200

## Summary

- Add a warning message to UI indicating how much the sell amount must be in order to be more likely to get matched.
- Warning is dependent of the sell amount.
- Add FAQ entry explaining the min fee constraint

### To define
- Sell amount warning threshold
- How to calculate given amount
- FAQ message
- Warning message

### Proposed texts
**Warning msg:** `Only trades that with a sell amount >= x TOKEN are likely to be settled. See FAQ`

**FAQ:**
_Q:_ ` What is the minimum fee per order?`
_A:_ `On the solvers owned/controlled by Gnosis, there's a min fee requirement orf the whole batch to be of X OWL, to make it economically viable. Orders might be disregarded if the fee generated does not contribute to reaching the min X OWL goal. Other competing solvers might implement different strategies where this constraint may or may not apply`

**Comment by josojo** [comment]
>What about orders between tokens like WETH or *BTC where it's not straight forward to infer the OWL relationship?
Do we need an additional quote query against OWL to be able to tell? (edited) 

The backend price estimation tool will give you the price of all tokens against OWL. 

> Where the x is calculated and TOKEN is replaced by with the current selltoken
How should this calculation be done?

It would be good if you guys could implement the same logic, as we will have in the backend, described here:


Here is one example:
```
subsidize_factor:=10
Gas price: 40 Gwei
OWL price per token: 240
=> min economical viable fee = 1.2 OWL
=> Handover to the solver: --min-avg-fee-per-order= 1.2/10
=> Min tradeable amount per token: 0.12 OWL *1000 * 3/2 / 240 (min_owl_fee * fee_factor * settlement_factor / price_of_token
```
The `settlement_factor` comes from the fact, that we also need to settle the fee order. Assuming that we have 2 orders in the batch being traded against each other and 1 fee order, the fee order will decrease the average_min_fee_per_order, and hence, the trading volume of the other orders need to compensate this and we scale their min. volume by the factor 3/2.

**Comment by josojo** [comment]
Could we get this issue tackled next week?

**Comment by alfetopito** [comment]
Sorry @josojo , was not able to get to it yet.
Will prioritize it for next release, yes.

**Comment by anxolin** [comment]
Does the 2 decimals thing affect this? 
I understand it affects the minimal trade, right?


> Warning msg: Only trades that with a sell amount >= x TOKEN are likely to be settled. See FAQ

I'm not sure if we should be that specific, given that other solvers can submit their own solutions. I would make it a bit more like:

> This is a small order. Please keep in mind that solvers might not include your order if it doesn't generate enough fee to pay their running costs.

**Comment by josojo** [comment]
> I'm not sure if we should be that specific, given that other solvers can submit their own solutions.

I think is valuable to tell the user about the min-trading volume, directly. Otherwise, they have to binary search it through the interface by changing the sellamount and checking the error message. This is information that users want to know in my opinion.

**Comment by anxolin** [comment]
I'm confused now. You are suggesting we should show it regardless of the sell volume? 
so in your opinion we should have a permanent warning with this?

Maybe we can do a quick zoom and see how is the best way

**Comment by Velenir** [comment]
_Estimation: 3_

**Comment by Velenir** [comment]
Closed by #1268
